packages/millicast-sdk/src/PeerConnection.js

```diff
@@ -43,11 +43,15 @@ export default class PeerConnection extends EventEmitter {
   /**
    * Instance new RTCPeerConnection.
    * @param {RTCConfiguration} config - Peer configuration.
+   * @param {Boolean} [autoInitStats = true] - True to initialize statistics monitoring of the RTCPeerConnection accessed via Logger.get(), false to opt-out.
    */
-  async createRTCPeer (config = null) {
+  async createRTCPeer (config = { autoInitStats: true }) {
```

Comment:
Probably worth checking - I wasn't seeing stats by default in `Logger.diagnose()` on the publishing side. maybe my setup was fudged, I dunno - but would be good to cross-check

Reply:
Logger.diagnose() works as expected for me after starting a stream in the publisher demo.  We can sync up at standup on this.